### ISSUE TITLE
Takes > 5 min for the client to be able to login

### ISSUE THREAD

**Issue by schappim** [ondevice/ondevice#18]
It takes > 5 min for the client to be able to use credentials to login. 
The error message provided until the credentials worked was: 

```2017/11/28 11:21:49 Fatal: Authentication failed: You need the '' permission to continue (keyId=1000)```

This is on MacOS 10.12.6 (16G29). 
Using Bash as the shell.

I don't think the problem was between the keyboard and the seat as I was copying and pasting the credentials...

**Comment by mreithub** [comment]
Hi, sry for your trouble.

I've just had a look at the API server logs and it seems the API server handling your initial requests was running an older software version that had a bug with `custom` keys (I put that one back in the loop to be prepared for any additional strain caused by the HN story).

The reason it worked a couple minutes later was that then your request was answered by the other server which had already been fixed.

I've updated the API server in question now.

Thanks for reporting.  
Things have been a little chaotic yesterday tbh but I hope you'll still give ondevice a chance ;)
